db/operations_test.go

```diff
@@ -108,18 +107,20 @@ func TestEscapeValues(t *testing.T) {
 			_, err = tx.ExecContext(ctx, insertStatement)
 			require.NoError(tt, err)
 
-			fmt.Println(insertStatement)
+			checkStatement := `select "col" from "test";`
+			row := tx.QueryRowContext(ctx, checkStatement)
+			var value string
+			err = row.Scan(&value)
+			require.NoError(tt, err)
+			require.Equal(tt, str, value, "Inserted value is not equal to the expected value")
 
 			selectStatement := `select * from "test";`
-			rows, err := tx.QueryContext(ctx, selectStatement)
+			_, err = tx.QueryContext(ctx, selectStatement)
 			require.NoError(tt, err)
```

Comment:
That's not an insert test, it's actually a variation of the previous test with now the column specified. So we gain nothing here.

And my comment was on `TestEscapeColumns` part for the insert, not the `TestEscapeValues`.